src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -265,8 +293,9 @@ void GcsServer::Stop() {
 
 void GcsServer::InitGcsNodeManager(const GcsInitData &gcs_init_data) {
   RAY_CHECK(gcs_table_storage_ && gcs_publisher_);
+  auto cluster_id = rpc_server_.GetClusterId();
```

Comment:
Here it's a copy, but later it's passed by reference and stored as reference. Anything wrong here?

Reply:
Made it inline